src/ParallelDataCollector.cpp

```diff
@@ -956,6 +970,45 @@ namespace splash
         dataset.close();
     }
 
+    CollectionType* ParallelDataCollector::readDataSetMeta(H5Handle h5File,
+            int32_t id,
+            const char* name,
+            const Dimensions dstBuffer,
+            const Dimensions dstOffset,
+            const Dimensions srcOffset,
+            Dimensions &sizeRead,
+            uint32_t& srcDims)
+    throw (DCException)
+    {
+        log_msg(2, "readDataSetMeta");
+
```

Comment:
```
 +--------+   +     +   +-+
 |        |   |   --+   |-|
 |        |   | |       |-+
 |        |   +-+--+    ++
 |        |   |    |
 +--------+   +    ++   ++

 XXXXX      XXXX       XXX
XX   XX  XXXX  XXX    XX XXX
      XXXX        XXXXX    XXXXX
```

But [asciiflow.com](http://asciiflow.com/) took even that joy away from us making it a less-then-a-few-months task :rabbit: